python_examples/gibraltar_example/gibraltar_inputs/gib_input.csv

```diff
@@ -1,4 +1,978 @@
 cell,microcell,location_x,location_y,household_number,place_number,Susceptible
+<<<<<<< HEAD
```

Comment:
You've left this in from a merge - and why did you need to change this file anyway ahaha